app/modules/individuals/resources.py

```diff
@@ -811,3 +811,64 @@ def get(self, individual):
         augmented_json = individual.augment_edm_json(result_json)
 
         return augmented_json
+
+
+@api.route('/validate')
+class FlatfileNameValidation(Resource):
+    # values passed in from flatfile are val/index pairs:
+    # [
+    #   ["Zebulon", 1],
+    #   ["Zebrucifer", 2],
+    #   ["Zebrelda", 300],
+    #   ["Zesus", 46]
+    # ]
+    def post(self):
```

Comment:
There's an issue here in that it's not using the ModuleAccess permissions check, but neither is the Individual creation itself so that's a separate issue that @colin and I need to talk about outside this PR.